src/scripts/main.js

```diff
@@ -1 +1,63 @@
 'use strict';
+
+const body = document.getElementById('document');
```

Comment:
There is no element with id 'document'. It's better to use document.body to select the body element.